### ISSUE TITLE
Add support for setting environment variables in a target.

### ISSUE THREAD

**Issue by jjzanin-SMS** [EpicGames/raddebugger#73]
It is useful to be able to specify environment variables for a target so that the program being debugged is launched under that environment. Something similar (if not exactly) like what Visual Studio has with the "Environment" and "Merge Environment" settings under "Debugging" in the project settings.

**Comment by MohitSethi99** [comment]
+1
I also use it to have `_NO_DEBUG_HEAP=1`, not sure it is by default here or not.

**Comment by jeffrbig2** [comment]
It feels like this is more useful when we have remote debugging, right? I mean, you start the debugger with a batch file or whatever anyway, so you can just add some "set=" to the front, and then hey, you can check that script in too.  

Certainly for remote debugging you need something like this, but for the current local stuff, this seems like it's better done in a script...?

**Comment by MohitSethi99** [comment]
Yeah it makes sense, I have no other use case as of now!

**Comment by jeffrbig2** [comment]
OK, we'll add it to the to-do section under remote debugging.  Thanks!

**Comment by jjzanin-SMS** [comment]
Yep, targeting my batch file that sets up the environment as opposed to targeting the executable works fine. It didn't occur to me to try that :-)

**Comment by forrestsmithfb** [comment]
> Certainly for remote debugging you need something like this, but for the current local stuff, this seems like it's better done in a script...?

I think it makes sense to include in the debugger. I'd like a nice little GUI that lets be define a variety of targets. Each target can have an exe, cmd args, working directory, and environment variables. 

I don't really want to have 10 different batch files for launching `raddbg.exe` in 10 different permutations. And if I'm bouncing between two or three I don't want to try and remember which instance of `raddbg.exe` was launched with which environment variables.

A batch file is a functional work around so it is "nice to have" rather than "blocking".

(fwiw I despise environment variables but Linux people hate me and want me to suffer so they rely on them extensively. and I often deal with Linuxy projects on Windows. grumble grumble.)

**Comment by mmozeiko** [comment]
For multi-target debugging I could see need to set different env vars for different targets. Even when running everything locally.

**Comment by jjzanin-SMS** [comment]
For my current case, I already had a batch file that set up the environment variables and then called the executable. In Visual Studio I set up the same environment variables and the target was the executable. In raddbg I could target the batch file itself and debug from there. Single-stepping lands you in cmd.exe, so you have to put a breakpoint in your program.

**Comment by randrew** [comment]
Really missing this one when I use raddbg instead of remedybg. In one of my current projects, I can't use a .bat file to launch the target program, and instead have to use a .bat file to launch raddbg. Having to juggle batch files to have the right environment variables set up, and remembering which target to launch in raddbg while remembering which bat file I last used to launch raddbg, is kinda prone to making mistakes and having things go wrong.

**Comment by ryanfleury** [comment]
Added in 0.9.16.

**Comment by jjzanin-SMS** [comment]
Thank you for adding this feature. Is there a way to have it resolve existing environment variables? For example:
PATH=%PATH%;c:/some/other/path
